### ISSUE TITLE
Very stale coverage baselines

### ISSUE THREAD

**Issue by mattmoor** [knative/test-infra#679]
In Serving a PR I just submitted got back:
![image](https://user-images.githubusercontent.com/2442466/55685124-83a85100-5907-11e9-95b8-992ceb79a411.png)

I'm wondering if whatever updates the coverage baselines is just broken because this file's existed for a while now...

/kind bug

**Comment by jessiezcc** [comment]
@steuhs, can u pls look into this?

**Comment by steuhs** [comment]
The baselines are stale because in recent runs, the unit test in github.com/knative/serving/pkg/activator/handler kept failing. We only use the most recent healthy run as baseline. I tried cloned a fresh copy of knative/serving and I see that test fail locally as well. I think the test need to be fixed @mattmoor

**Comment by mattmoor** [comment]
We're talking like a month old, there's no way we've had a failing unit test for that long.

Here's a 100x run on master as of now:
```
$ go test -count=100 ./pkg/activator/handler/
ok      github.com/knative/serving/pkg/activator/handler        11.111s
```

With coverage (what I run within emacs):
```
go test -covermode=count -coverprofile=/tmp/count.out github.com/knative/serving/pkg/activator/handler/...
ok      github.com/knative/serving/pkg/activator/handler        0.341s  coverage: 97.1% of statements
```

The baselines are so old at this point, we may as well turn covbot off, so please get this fixed.

Also: why did it take me raising this as stale coverage for us to uncover this?  @jessiezcc can we add this to some sort of triage the oncall monitors and takes action on?

**Comment by jessiezcc** [comment]
Matt is right. The coverage for this file shows between 96%-100% since April.8th on [testgrid](https://testgrid.knative.dev/knative-serving#coverage). 
This should be covered by monitoring spec @steuhs, pls make sure you add this scenario into functional spec. And before automation is in place, let's add a section to oncall playbook for manual triage. 
/cc @srinivashegde86 who is the project lead for monitoring/metrics.

**Comment by jessiezcc** [comment]
@steuhs, can you pls update baseline asap for now?

**Comment by steuhs** [comment]
added a temporary manual fix. the baseline should be up to date now.

This is a link showing the failure of github.com/knative/serving/pkg/activator/handler, mentioned before.
https://gubernator.knative.dev/build/knative-prow/logs/post-knative-serving-go-coverage/1123312530802872321/?log#log

**Comment by steuhs** [comment]
fixed by #728
